packages/react-sdk/src/components/link.tsx

```diff
@@ -1,13 +1,29 @@
+import { Link as RemixLink } from "@remix-run/react";
 import { forwardRef, type ElementRef, type ComponentProps } from "react";
 
-const defaultTag = "a";
+const isAbsoluteUrl = (href: string) => {
+  // http:, mailto:, tel:, etc.
+  return /^[a-z]+:/i.test(href);
+};
+
+export const renderLink = (
+  href: string,
+  props: Omit<ComponentProps<"a">, "href">,
+  ref: React.ForwardedRef<HTMLAnchorElement>
+) => {
+  return isAbsoluteUrl(href) ? (
+    <a {...props} href={href} ref={ref} />
+  ) : (
+    <RemixLink {...props} to={href} ref={ref} />
```

Comment:
since we now hvae src/spp/custom-components in sdk package, we should clearly separate react components and remix components, so we can later move the app-level components and logic out of SDK

In this case it means the link link components in  src/components can stay dumb and we wrap it into that component that uses RemixLink

Does this make sense to you?

Reply:
Yeah, was also thinking about this